<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmConnexion</class>
 <widget class="QDialog" name="frmConnexion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connexion au profil</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="lblPseudo">
        <property name="text">
         <string>Identifiant</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPseudo"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblPwd">
        <property name="text">
         <string>Mot de passe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPassword"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="btCreation">
      <property name="text">
       <string>Cr√©er un profil</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btConnexion">
      <property name="text">
       <string>Se connecter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
